packages/test/test-version-utils/src/describeCompat.ts

```diff
@@ -42,10 +42,13 @@ function createCompatSuite(
                     );
                     Object.defineProperty(this, "__fluidTestProvider", { get: () => provider });
                 });
-                tests.bind(this)((reset: boolean = true) => {
+                tests.bind(this)((reset: boolean = true, syncSummarizer: boolean = false) => {
                     if (reset) {
                         resetAfterEach = true;
                     }
+                    if (syncSummarizer) {
```

Comment:
@anthony-murphy @markfields I could not find a better way to do tell the test object provider to create a loader container tracker with this flag enabled. Any suggestions?

Reply:
there probably isn't, but i'd prefer this be an object if we want to add to it, as continuing to add optional parameters won't scale.

Reply:
Updated the params to be an object.